nanoFirmwareFlasher/FirmwarePackage.cs

```diff
@@ -146,12 +146,23 @@ protected async System.Threading.Tasks.Task<ExitCodes> DownloadAndExtractAsync()
 
             if (fwFiles.Any())
             {
-                // get file creation date (from the 1st one)
-                if ((DateTime.UtcNow - File.GetLastWriteTimeUtc(fwFiles.First().FullName)).TotalHours < 4)
+                //// get file creation date (from the 1st one)
+                //if ((DateTime.UtcNow - File.GetLastWriteTimeUtc(fwFiles.First().FullName)).TotalHours < 4)
+                //{
+                //    // fw package has less than 4 hours
+                //    // skip download
+                //    skipDownload = true;
+                //}
+
+                if (!string.IsNullOrEmpty(_fwVersion))
```

Comment:
Why are these checks necessary? Isn't this just a matter of _not_ checking the file date? 🤔

Reply:
I have removed the whole Block and does not affect the running of the program.